deletePatientData/0.92/deletePatientData.sh

```diff
@@ -0,0 +1,96 @@
+
+GITHUB_URL="https://raw.githubusercontent.com/Bahmni/bahmni-scripts/master/deletePatientData"
```

Comment:
needs to be ... "../deletePatientData/0.92". Check the subsequent wget commands.